### ISSUE TITLE
Filter messages by tag

### ISSUE THREAD

**Issue by eopo** [binwiederhier/ntfy#1087]
:bulb: **Idea**
Currently, tagging messages with text tags does not do anything different, than just including the tags in the message body.
A user would greatly benefit from being able to filter messages by tag, ideally by just clicking on a tag on an existing message.
In a use case of server monitoring with the server name, data center name or similar as tags, a user could filter messages by server and therefore easily find out, if a server is back up after sending an error or if a high temperature alert is recurring.
Styling tags in a button fashion could improve the usability by implying, that tapping them does something.

:computer: **Target components**
All clients

**Comment by wunter8** [comment]
Do you know about this? https://docs.ntfy.sh/subscribe/api/#filter-messages

**Comment by eopo** [comment]
Yeah, but unfortunately at least the iOS-App and the Web App don't support it. That's why I mentioned clients as target components.

**Comment by wunter8** [comment]
The project creator decided not to implement this in the past: https://github.com/binwiederhier/ntfy/issues/703

I'm not sure if anything has changed
